core/src/Cabana_Core.hpp

```diff
@@ -49,4 +49,13 @@
 #include <Cabana_Experimental_NeighborList.hpp>
 #endif
 
+#ifdef Cabana_ENABLE_SILO
+#include <Cabana_SiloParticleOutput.hpp>
+#endif
+
+#ifdef Cabana_ENABLE_HDF5
+#include <Cabana_HDF5ParticleOutput.hpp>
+#include <xdmf_header_info.h>
```

Comment:
I'll append it. 
I could do a separate PR later for reading if you don't mind there not being a valid check for the I/O. 
I don't need to track the development history of the commits so it would be better to squash merge it.

Reply:
fixed.